### ISSUE TITLE
How to make radio button selected/checked using yaml?

### ISSUE THREAD

**Issue by ayumi-cloud** [octobercms/docs#433]
How to make radio button checked using yaml?

There is no answer to this in the doc's found here: https://octobercms.com/docs/backend/forms#field-radio

However, looking at the code in the main repo found here: https://github.com/octobercms/october/blob/master/modules/backend/widgets/form/partials/_field_radio.htm#L22

Says that October is looking for: `$field->isSelected($value)`

Questions:

1. Is there a way to make a radio option `selected / checked` via yaml.

2. If so then can we add the information to the doc's.

3. I believe I had the same issue with checkboxes a while ago and suggested we should update the doc's also for making them `checked` via yaml.

Cool, I'll wait for a reply (not going to tag admins as I know everyone's busy).

**Comment by ayumi-cloud** [comment]
~No need reply, we already found solution and will upload the pr in a few hours after we had a long lunch.

**Comment by bennothommo** [comment]
@ayumi-cloud Try `default: true` in the YAML config.

**Comment by ayumi-cloud** [comment]
@bennothommo Hi thanks for your suggestion, sorry if I'm being a bit thick! I'm not sure how the `default: true` would work in this situation.

I give more information and see what your thoughts are on this:

```yaml
example:
    type: radio
    options:
        one: One
        two: Two
```

Let's say for example we want the following to be setup from the start:

![image](https://user-images.githubusercontent.com/57409060/75174529-a2ceed80-5728-11ea-8a90-c0232fcfb64b.png)

We are under the impression that this can't be done through yaml at the moment. Instead we set this up using a seeder file instead.

Please correct me if I'm wrong!

So I was thinking of adding this information to the doc's.

**Comment by LukeTowers** [comment]
@ayumi-cloud radio buttons work the same as dropdowns
```yaml
example:
    type: radio
    default: one
    options:
        one: One
        two: Two
```

**Comment by ayumi-cloud** [comment]
@LukeTowers 

Ah now I see and get what you both are saying! Sorry language barrier when I read the:

```
default | specifies the default value for the field.
```

To me it makes better sense saying something like:

```
default | specify the 'value' field you want to make selected or checked on page load.
```

**Comment by bennothommo** [comment]
@ayumi-cloud Sorry my bad, I misread your original post to mean "checkbox", not radios. @LukeTowers' answer is the correct one here.

**Comment by LukeTowers** [comment]
Fixed in #434
